### PR TITLE
ts-refactor - renamed files to ts

### PR DESCRIPTION
**Pull request by ujahio**

### CODE REVIEW

**Review by rosiehsmith — APPROVED** [review]
Nice! Love the small PR as well, makes it easier to see what changed.

File: shared/src/business/useCases/trialSessions/generatePrintableTrialSessionCopyReportInteractor.ts [hunk 1]
```diff
@@ -29,13 +29,13 @@ export const generatePrintableTrialSessionCopyReportInteractor = async (
...
     userHeading,
   }: {
-    showCaseNotes: boolean;
-    filters: string[];
-    formattedCases: TCase[];
-    formattedTrialSession: TTrialSessionData;
-    sessionNotes: string;
-    sort: string;
-    userHeading: string;
+    filters?: TPrintableTableFilters;
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/trialSessions/generatePrintableTrialSessionCopyReportInteractor.ts]: Why do are all these arguments optional?

File: shared/src/notifications/sendNotificationToConnection.ts [hunk 2]
```diff
@@ -6,10 +6,15 @@
...
  */
-exports.sendNotificationToConnection = async ({
+export const sendNotificationToConnection = async ({
   applicationContext,
   connection,
   messageStringified,
+}: {
+  applicationContext: IApplicationContext;
+  connection: TConnection;
+  deleteGoneConnections?: boolean;
```
Comment (rachelschneiderman) [inline shared/src/notifications/sendNotificationToConnection.ts]: I don't see that `deleteGoneConnections` is an argument to this function, can it be removed?

**Review by rachelschneiderman — COMMENTED** [review]
Wonderfully small PR, well done Josh!!!! 👏 👏 👏

File: shared/src/business/useCases/trialSessions/generatePrintableTrialSessionCopyReportInteractor.ts (hunk 1 above)
Comment (ujahio) [reply shared/src/business/useCases/trialSessions/generatePrintableTrialSessionCopyReportInteractor.ts]: I have made the adjustments to remove the options. All usage of this interactor passes all of these props so they are required. I made the adjustments to the test to pass all of the props as well.

File: shared/src/notifications/sendNotificationToConnection.ts (hunk 2 above)
Comment (ujahio) [reply shared/src/notifications/sendNotificationToConnection.ts]: Removed.

**Pull request merged**